PKGBUILD

```diff
@@ -40,11 +41,11 @@ source_armv7h=("caddy.tar.gz::${_url_prefix}&arch=arm" "caddy-systemd-service.pa
 source_aarch64=("caddy.tar.gz::${_url_prefix}&arch=arm64" "caddy-systemd-service.patch")
 
 prepare() {
-  patch -p0 -i caddy-systemd-service.patch
+  msg2 "Patching systemd service file"
```

Comment:
msg2 ist a formated echo in a short answer. looks nicer. :)
And yes, the documentation for this script is not exist. :(
But it will used in most packages.